### ISSUE TITLE
Increase number of power lines to 4

### ISSUE THREAD

**Issue by timoo54** [openinframap/openinframap#150]
On the power layer currently a maximum of 3 circuits is rendered on any given line.
Power lines with 4 circuits are quite common e.g. 2 x 380kv + 2 x 110kv on the same line. e.g.:
https://commons.wikimedia.org/wiki/File:20200416_hoogspanningsleiding_door_het_Valtherbos.jpg
Currently this is rendered with 2 x 380kv and and only 1 x 110kv. Also the description ends with voltage 3.

I suggest to increase the number of displayed circuits to 4.
There are also lines with 6 circuits, however these are quite rare.

**Comment by Hiausirg** [comment]
Not in the US, i'm almost done with all lines in South Carolina, and i've seen more than 3 only for like 100 metres in downtown Charleston. And even three are very rare. But, i can also tell from personal experience: VERY common in Europe, so strong support.
And if we're already increasing limits: The maximum displayed voltages on substations are also capped at max 3 - should be raised, for substations like https://www.openstreetmap.org/way/120874146 ([OIM](https://openinframap.org/#16.06/35.223673/-81.188774/L,O,P,T,W))

**Comment by flacombe** [comment]
+1 In France there are a few miles of 5-circuits power lines and significantly more of 4-circuits.
In Switzerland, Germany, Austria there even more circuits on a single line with 16.7 Hz network for railways.

To me, voltage=* on substation could only contains the highest value.

Voltage isn't a proper property for substation itself, but contained equipments.
Anyway, it's a noticeable effort to populate this key with every voltages found in the facility, well done!

**Comment by tntchn** [comment]
Agreed. I saw many giant towers with 4 circuits in Taiwan. Recently the network company are building more underground tunnels that are able to place 6 circuits in one tunnel. 
And if we decide to increase the displayed lines to 4+ I think the gap between lines can be reduced to keep the map looks good.

**Comment by Gazer75** [comment]
In densely populated and warm climate areas it is not uncommon to have 4-6 circuits on same tower.
I've seen quite a few stretches in Germany that have at least 4.
